express-zod-api/tsdown.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "tsdown";
+import { readFile } from "node:fs/promises";
+
+const { version } = JSON.parse(await readFile("./package.json", "utf8"));
+
+export default defineConfig({
+  entry: "src/index.ts",
+  minify: true,
```

Comment:
this feature is risky
https://tsdown.dev/options/minification#minification

might be better to remove it